Makefile

```diff
@@ -83,7 +83,10 @@ init: default
 ### run:              Start the apisix server
 .PHONY: run
 run: default
-ifeq ("$(wildcard logs/nginx.pid)", "")
+ifeq ("$(findstring /root, ${PWD})", "/root")
```

Comment:
We need a thorough solution, not a temporary patch

Reply:
Ok, got it.

Reply:
updated.